src/order/filter/basic.lean

```diff
@@ -614,6 +634,19 @@ protected def frequently (p : α → Prop) (f : filter α) : Prop := ¬∀ᶠ x
 
 notation `∃ᶠ` binders ` in ` f `, ` r:(scoped p, filter.frequently p f) := r
 
+lemma frequently_iff {f : filter α} {P : α → Prop} :
+  (∃ᶠ x in f, P x) ↔ ∀ {U}, U ∈ f → ∃ x ∈ U, P x :=
```

Comment:
Note that we already have `frequently_iff_forall_eventually_exists_and`.

Reply:
This one is intended as moving away from eventually/frequently. But indeed I didn't notice the other one. I moved mine after the existing one, and use the existing one (which requires quite a bit of bureaucracy).